package.json

```diff
@@ -121,7 +124,6 @@
     "react-dom": "^17.0.1",
     "socksv5": "0.0.6",
     "style-loader": "^1.2.1",
-    "ts-loader": "^8.0.3",
```

Comment:
Webpack 5 should be quite stable now. And it has also perf enhancements: https://webpack.js.org/migrate/5/
Might make sense for a drive-by/follow-up PR?